pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -156,7 +156,15 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 
 	// provide any config overrides
 	cfg.NodeName = options.NodeName
-	cfg.StreamingConnectionIdleTimeout = 5 * time.Minute // TODO: should be set
+	streamingConnectionIdletimeout, ok := options.KubeletArguments["streaming-connection-idle-timeout"]
+	if ok && len(options.KubeletArguments["streaming-connection-idle-timeout"]) > 1 {
```

Comment:
change the default in `kubelet/app/server.go` from 0 to the current value of `StreamingConnectionIdleTimeout`, like this:
```
fs.DurationVar(&s.StreamingConnectionIdleTimeout, "streaming-connection-idle-timeout", s.StreamingConnectionIdleTimeout, "Maximum time a streaming connection can be idle before the connection is automatically closed.  Example: '5m'")
```
and open an upstream PR for that